src/lib/server/email/dispatch.ts

```diff
@@ -71,6 +71,34 @@ export class NotificationDispatcher implements Loggable {
     return job;
   }
 
+  async #sendBulkNotificationRequest(notifRequests: Notification[]) {
+    const requests = await this.#db.insertNotificationsBulk(notifRequests);
+
+    this.#logger.info('new notification requests bulk received', { requests });
+
+    const jobs = requests.map(({ id }) => {
+      return {
+        name: id,
+        data: { requestsId: id },
+        opts: {
+          jobId: id,
+          removeOnComplete: true,
+          attempts: 3,
+          backoff: {
+            type: 'fixed',
+            delay: 3000,
+          },
+        },
+      };
+    });
+
+    const insertedJobs = await this.#queue.addBulk(jobs);
+
+    this.#logger.info('new jobs created', { insertedJobs });
```

Comment:
I think only printing the job IDs is enough. Otherwise, our logs would be polluted with a bunch of job info objects.